spec.md

```diff
@@ -146,32 +138,46 @@ that contains both context and data).
   * REQUIRED
   * MUST be a non-empty string
 
-### source
-* Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
-* Constraints:
-  * REQUIRED
-  * MUST contain at least one non-empty sub-property.
-
-### source-type
+### source-authority
```

Comment:
Does this source-authority mean URI authority, i.e any event attribute of type "URI" must be compliant with this authority?